Amy_Core_Main.launch

```diff
@@ -12,5 +12,5 @@
 <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="de.unistuttgart.iaas.amyassist.amy.core.Main"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="amy-core"/>
 <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:amy}"/>
+<stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:Amy}"/>
```

Comment:
is this change needed?

Reply:
It didn't work for me so I changed it back and now it works again